arkane/qchem.py

```diff
@@ -270,29 +270,20 @@ def loadEnergy(self, frequencyScaleFactor=1.):
         """
         E0 = None
     
-        f = open(self.path, 'r')
-        line = f.readline()
-        while line != '':
-    
-            if 'Final energy is' in line:
-                E0 = float(line.split()[3]) * constants.E_h * constants.Na
-                logging.debug('energy is {}'.format(str(E0)))
-            
-#            elif 'Zero point vibrational energy' in line:
-                #Qchem's ZPE is in kcal/mol
-#                ZPE = float(line.split()[4]) * 4184
-#                scaledZPE = ZPE * frequencyScaleFactor
-#                print 'ZPE is ' + str(ZPE)
-            # Read the next line in the file
+        with open(self.path, 'r') as f:
             line = f.readline()
-    
-        # Close file when finished
-        f.close()
+            while line != '':
```

Comment:
While you're making it more pythonic, the following is a lot simpler if you're going to read the whole file:
```
with open(self.path, 'r') as f:
    for line in f:
        # Do something
```

Reply:
Thanks for the comment!